website/src/pages/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import styles from './styles.module.css';
+
+const features = [
+  {
+    title: 'Partition data into a hexagonal grid',
+    imageUrl: 'images/icon-high-precision.svg',
+    description: (
+      <>
+        H3 has an easy API for indexing coordinates into a hexagonal, global grid.
+      </>
+    ),
+  },
+  {
+    title: 'Hierarchical grid system',
+    imageUrl: 'images/icon-layers.svg',
+    description: (
+      <>
+        Easy, bitwise truncation to coarser, approximate cells, along with area compression/decompression algorithms.
+      </>
+    ),
+  },
+  {
+    title: 'Fully global',
+    imageUrl: 'images/icon-chart.svg',
+    description: (
+      <>
+        Along with twelve pentagons, the entire world is addressable in H3, down to square meter resolution.
+      </>
+    ),
+  },
+];
+
+function Feature({imageUrl, title, description}) {
+  const imgUrl = useBaseUrl(imageUrl);
+  return (
+    <div className={clsx('col col--4', styles.feature)}>
+      {imgUrl && (
+        <div className="text--center">
+          <img className={styles.featureImage} src={imgUrl} alt={title} />
+        </div>
+      )}
+      <h3>{title}</h3>
+      <p>{description}</p>
+    </div>
+  );
+}
+
+export default function Home() {
+  const context = useDocusaurusContext();
+  const {siteConfig = {}} = context;
+  return (
+    <Layout
+      title={`${siteConfig.title}`}
+      description="Hexagonal hierarchical geospatial indexing system">
```

Comment:
Not required right now, but we should make sure we have appropriate metadata in the rendered page to play well with search engines and social media embedding